controllers/providers/aws/aws.go

```diff
@@ -201,6 +201,44 @@ func (w *AwsWorker) DeleteScalingGroup(name string) error {
 	return nil
 }
 
+func (w *AwsWorker) SetSuspendProcesses(name string, processesToSuspend []string) error {
+
+	var scalingProcesses []*string
+
+	for _, process := range processesToSuspend {
+		scalingProcesses = append(scalingProcesses, aws.String(process))
+	}
+
+	input := &autoscaling.ScalingProcessQuery{
+		AutoScalingGroupName: aws.String(name),
+		ScalingProcesses:     scalingProcesses,
```

Comment:
You can probably just use `aws.StringSlice(processesToSuspend)` and remove the iteration in the top